examples/fine-tuning/alignment-handbook/train.dstack.yaml

```diff
@@ -0,0 +1,39 @@
+type: task
+
+python: "3.11"
+
+env:
+  - HUGGING_FACE_HUB_TOKEN
+  - CONFIG_PATH
```

Comment:
Do we need it as an environment variable? I suppose it is always `config.yml`, right? IMO to simplify, I'd hardcode it.

Reply:
For the sake of simplicity of this as a tutorial, I removed `CONFIG_PATH` and `NUM_GPU` environment variables. However, I do think it is common not to hardcode `yaml` file path. It is very common to have lots of YAMLs with different configurations to run multiple experiments with different hyper-params.